Dockerfile

```diff
@@ -55,7 +55,7 @@ RUN apk update && \
     python3 -m venv $VIRTUAL_ENV
 
 COPY requirements.lock.txt requirements.txt
-ARG PIP_EXTRA_INDEX_URL="https://artifactory.algol60.net/artifactory/csm-python-modules/simple"
+ARG PIP_EXTRA_INDEX_URL="https://artifactory.algol60.net/artifactory/csm-python-modules/simple https://artifactory.algol60.net/artifactory/csm-python-modules/unstable"
```

Comment:
link to unstable

Reply:
As we were testing unstable build we kept it. Will drop the second commit which I used to add unstable one once Ranga finishes all the testing.